docs/chainlink-keepers/best-practices.md

```diff
@@ -8,81 +8,86 @@ whatsnext:
     'Network Overview': '/docs/chainlink-keepers/overview/',
   }
 ---
-Chainlink Keepers are a powerful tool for smart contract automation. These are some useful patterns and best practices that you can employ within your Keeper-compatible smart contract.
+Chainlink Keepers are a powerful tool for smart contract automation. This guide demonstrates some useful patterns and best practices that you can employ within Keeper-compatible smart contracts.
 
-It is important to understand the gas limits for on-chain execution and off-chain computation when developing your Keeper-compatible smart contract - you can review that in [How Chainlink Keepers Work](../overview/).
+It is important to understand the gas limits for on-chain execution and off-chain computation when developing your Keeper-compatible smart contract. To learn more about the economics of Chainlink Keepers, read [How Chainlink Keepers Work](../overview/).
 
 ## Use `checkData` and `performData` arguments
```

Comment:
Current H2 Headings:

- Use `checkData` and `performData` arguments
- Modifiers
- Patterns to Consider
- Testing
- Gas Limits

I think a better set would be:

- Functions
- Modifiers
- Useful Patterns
- Gas Limits
- Testing

Reply:
Updated in latest push.